src/store.js

```diff
@@ -0,0 +1,11 @@
+import { applyMiddleware, createStore } from "redux"
+import logger from "redux-logger"
+import thunk from "redux-thunk"
+import promise from "redux-promise-middleware"
+import allReducers from "./reducers"
+
+const middleware = applyMiddleware(promise(), thunk, logger())
```

Comment:
Use `process.env.NODE_ENV === 'production'` to determine whether or not to include `logger()`. You don't want to include `logger()` when you're building the react code for production.